lib/testing/integration/ssh_test.go

```diff
@@ -69,22 +69,16 @@ func (s *IntegrationSSHSuite) TestSSH() {
 	err = cmd.Start()
 	require.NoError(t, err)
 
-	_, err = stdinPipe.Write([]byte("whoami\n\r"))
```

Comment:
Is `whoami` not available on one of our platforms?

Reply:
It is available.
I changed this in order to simplify the check we do.

Depending on the prompt/shell configuration it could print one of the following:
- `\r<user>\r\n`
- `\n<user>\r\n`

So, instead of only matching for the `<user>\r\n` which would relax the check (this string could appear in the prompt), I changed this to print the user with a prefix and match on that.